Add docs for Update-IdentitySourceCertificates in Runcommand (#119950)
Azure/azure-content: articles/azure-vmware/configure-identity-source-vcenter.md

> [!WARNING]
> If you don't provide a value for **DomainName**, all external identity sources are removed. Run the cmdlet Update-IdentitySourceCredential only after the password is rotated in the domain controller.

## Renew existing certificates for LDAPS identity source

1. Renew the existing certificates in your domain controllers.

> 1. Optional: If the certificates are stored in default domain contollers, this step is optional. Leave SSLCertificatesSasUrl parameter blank and the new certificates will be downloaded from the default domain controllers and update them in vCenter automatically. If choose to not use the default way, please [export the certificate for LDAPS authentication](#to-export-the-certificate) and [upload the LDAPS certificate to blob storage and generate an SAS URL](#upload-the-ldaps-certificate-to-blob-storage-and-generate-an-sas-url-optional). Save the SAS URL for the next step.

Reviewer: ```suggestion
1. Optional: If the certificates are stored in default domain controllers, this step is optional. Leave the SSLCertificatesSasUrl parameter blank and the new certificates will be downloaded from the default domain controllers and updated in vCenter automatically. If you choose to not use the default way, [export the certificate for LDAPS authentication](#to-export-the-certificate) and [upload the LDAPS certificate to blob storage and generate an SAS URL](#upload-the-ldaps-certificate-to-blob-storage-and-generate-an-sas-url-optional). Save the SAS URL for the next step.
```

1. Select **Run command** > **Packages** > **Update-IdentitySourceCertificates**.

> 1. Provide the required values and the new SAS URL(optional), and then select **Run**.

Reviewer: ```suggestion
1. Provide the required values and the new SAS URL (optional), and then select **Run**.
```

   | **Field** | **Value** |
   | --- | --- |
   | **DomainName***  |  The FQDN of the domain, for example **avslab.local**.  |

>    | **SSLCertificatesSasUrl(optional)**  | A comma-delimited list of SAS path URI to Certificates for authentication. Ensure permissions to read included. To generate, place the certificates in any storage account blob and then right click the cert and generate SAS. If the value of this field is not provided from a user, the certificates will be downloaded from the default domain controllers.  |

Reviewer: ```suggestion
   | **SSLCertificatesSasUrl (optional)**  | A comma-delimited list of SAS path URI to Certificates for authentication. Ensure permissions to read are included. To generate, place the certificates in any storage account blob and then right-click the cert and generate SAS. If the value of this field isn't provided by a user, the certificates will be downloaded from the default domain controllers.  |
```
